### ISSUE TITLE
[Bug]: Adding a local pack using `cpackget` errors out when loading in dependent packages

### ISSUE THREAD

**Issue by automatiek** [Open-CMSIS-Pack/cmsis-toolbox#118]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Running `cpackget` on a local pack I get following error:

```
cpackget add gestures-dataset-cmsis-package-v313/EdgeImpulse.Gestures_Dataset.313.0.0.pack --force-reinstall
I: Adding pack "gestures-dataset-cmsis-package-v313/EdgeImpulse.Gestures_Dataset.313.0.0.pack"
I: Extracting files to /Users/arjan/.cache/arm/packs/EdgeImpulse/Gestures_Dataset/313.0.0...
I: 100% |███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| (12/12, 1840 it/s)        
I: Package requirements not satisfied - installing ARM::CMSIS-DSP@1.11.0 ARM::CMSIS-NN@4.0.0 
W: Could not resolve version modifier
I: Downloading ARM.CMSIS-DSP.1.15.0.pack...
I: 100% |███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| (8.1/8.1 MB, 6.1 MB/s)        

I: Adding pack ARM.CMSIS-DSP.
E: The latest release (1.15.0) in pack's pdsc (ARM.CMSIS-DSP.pdsc) does not match pack version ""
E: pack version is not the latest in the pdsc file
```

Here it looks like version information is not added correctly. When installing the packs manually with `cpackget add ARM::CMSIS-DSP@1.11.0`, installing works as expected

### Steps To Reproduce

using cmsis toolbox version 2.3.0

### Affected tools

- [ ] cbuild
- [ ] cbuild-gen
- [X] cpackget
- [ ] csolution
- [ ] packchk
- [ ] svdconv

### Affected operating systems

- [ ] Windows amd64
- [ ] Windows arm64
- [ ] Linux amd64
- [ ] Linux arm64
- [ ] MacOS amd64
- [X] MacOS arm64

### Environment

MacOS 14.2.1 (23C71)

**Comment by jkrech** [comment]
As far as I can tell this is the same problem as reported and fixed here
https://github.com/Open-CMSIS-Pack/cpackget/issues/283

As this is a regression, are you able to revert to CMSIS-Toolbox 2.2.1 until there is a patch release available?

**Comment by automatiek** [comment]
Hi @jkrech, thanks for your quick reply. 

It is indeed the same issue, and able to install the tools correctly using CMSIS-Toolbox 2.2.1. 

Feel free to close the issue, or keep it open for visibility.

**Comment by francovaro** [comment]
@jkrech it seems solved in 2.4.0

**Comment by jkrech** [comment]
Fixed in CMSIS-Toolbox 2.4.0
